apps/src/p5lab/AnimationPicker/AnimationPickerListItem.jsx

```diff
@@ -1,17 +1,17 @@
 /** @file A clickable item in the scroll area of the animation picker */
 import PropTypes from 'prop-types';
 import React from 'react';
-import Radium from 'radium'; // eslint-disable-line no-restricted-imports
-import color from '@cdo/apps/util/color';
 import {PlayBehavior} from '../constants';
 import * as shapes from '../shapes';
 import AnimationPreview from './AnimationPreview';
+import style from './animation-picker-list-item.module.scss';
+
+import classNames from 'classnames';
 
 const THUMBNAIL_SIZE = 105;
```

Comment:
I think you can remove this line and the one below it too?

Reply:
@megcrenshaw These constants are still used below at L76-77 to pass the height and width for the AnimationPreview component.
These values are set here and within the module, which doesn't seem great now that you mention it. I wonder how we can have one source of truth for thumbnail size across both components.

Reply:
Ugh. Just remembered that `$thumbnail-size: 105px;` while `const THUMBNAIL_SIZE = 105;`
I can't just export the pixel value because the other component that also uses the values (AnimationPreview) is expecting a number for these properties.

Reply:
@megcrenshaw (cc @madelynkasula)
This worked (https://github.com/code-dot-org/code-dot-org/pull/47478/commits/52921b50c006dd9b9f7432b1ed018ad0b71d237e), but it did hit [linting errors](https://codedotorg.slack.com/archives/C03MZ259SUU/p1661264943149599?thread_ts=1660840277.341799&cid=C03MZ259SUU). I ended up doing a `git commit --no-verify` to get around this, but I'd like to confirm that this is okay before merging something that might just be caught later in the pipeline any way.